lib/ref_arch_setup/bolt_helper.rb

```diff
@@ -91,6 +93,61 @@ def self.run_task_with_bolt(task, params, nodes)
       return success
     end
 
+    # Run a plan with bolt on given nodes
+    #
+    # @author Sam Woods
+    #
+    # @param task [string] Task to run on nodes
+    # @param params [hash] task parameters to send to bolt
+    # @param nodes [string] Host or space delimited hosts to run task on
+    # @param modulepath [string] The modulepath to use when running bolt
+    #
+    # @return [string] The output from the bolt run
+    def self.run_plan_with_bolt(plan, params, nodes, modulepath = RAS_MODULE_PATH)
+      params_str = ""
+      params_str = params_to_string(params) unless params.nil?
+      command = "bolt plan run #{plan} #{params_str}"
+      command << " --modulepath #{modulepath} --nodes #{nodes}"
+      command << bolt_options_string
+      puts "Running: #{command}"
+      output = `#{command}`
+      puts "Output was: #{output}"
+
+      success = $?.success? # rubocop:disable Style/SpecialGlobalVars
+      puts "Exit status was: #{$?.exitstatus}" # rubocop:disable Style/SpecialGlobalVars
+      raise "ERROR: bolt plan failed!" unless success
+
+      return output
```

Comment:
Should we chance run_task_with_bolt to match this?  Not sure there is a reason they need to be different.

Reply:
Discussed